packages/app-builder-lib/src/platformPackager.ts

```diff
@@ -608,13 +608,16 @@ export abstract class PlatformPackager<DC extends PlatformSpecificBuildOptions>
   getResourcesDir(appOutDir: string): string {
     if (this.platform === Platform.MAC) {
       return this.getMacOsResourcesDir(appOutDir)
-    } else if (isElectronBased(this.info.framework)) {
+    }
+    if (isElectronBased(this.info.framework)) {
       return path.join(appOutDir, "resources")
-    } else {
-      return appOutDir
     }
+    return appOutDir
   }
 
+  public getMacOsElectronFrameworkResourcesDir(appOutDir: string): string {
+    return path.join(appOutDir, `${this.appInfo.productFilename}.app`, "Contents", "Frameworks", "Electron Framework.framework", "Resources")
```

Comment:
I think `Electron Framework.framework` may change name if a custom electron distributable is compiled with a different `BRANDING` file.

As such, I think we may need to use: https://github.com/electron-userland/electron-builder/blob/3d65267a6c53ca824f70e5b0f5d8f4ba8be38237/packages/app-builder-lib/src/electron/ElectronFramework.ts#L29-L34

Reply:
Good catch.